src/core/EventBus.js

```diff
@@ -49,11 +51,13 @@ function EventBus() {
 
         var handler = {
             callback: listener,
-            scope: scope
+            scope: scope,
+            priority: priority !== undefined ? priority : LOW_PRIORITY
         };
 
         handlers[type] = handlers[type] || [];
         handlers[type].push(handler);
+        handlers[type].sort(function (a, b) {return a.priority - b.priority;});
```

Comment:
I do agree on clearing it up as to how to use it and what it means to the events with docs and const.  Any ideas.  We can clean this up for sure before we ship it ;) Thanks again for taking the time to do a nice review.  Very helpful and appreciated!